src/components/text-input/text-input.config.js

```diff
@@ -20,10 +20,18 @@ module.exports = {
     },
     {
       name: 'toggle password visibility',
-      label: 'Text Input (toggle password visibility)',
+      label: 'Password Input (toggle password visibility)',
```

Comment:
I think we could just leave this as `label: 'Password Input'` We don't add additional descriptors to any of the other components. What do you think?

Reply:
yeah that's probably better @alisonjoseph, I've changed it